code/_onclick/item_attack.dm

```diff
@@ -110,4 +110,4 @@
 		visible_message("<span class='warning'>[I] bounces harmlessly off of [src].</span>",\
 					"<span class='warning'>[I] bounces harmlessly off of [src]!</span>")
 	else
-		return ..()
\ No newline at end of file
+		return ..()

```

Comment:
I've been using Sublime Text and it only just occurred to me to look for a plugin that can solve this problem. Agh!!